extensions/rich_text_components/Skillreview/directives/oppia-noninteractive-skillreview.directive.ts

```diff
@@ -55,9 +55,15 @@ angular.module('oppia').directive('oppiaNoninteractiveSkillreview', [
             // instance, then open the concept card modal. To determine if the
             // RTE is inside a CKEditor instance, check if the offsetParent
             // element contains the data attribute ckeWidgetId.
-            if (event.currentTarget.offsetParent.dataset.ckeWidgetId) {
+            // Also check if the copy mode is enabled within the CKEditor
```

Comment:
To check whether copy mode is enabled or not, you'll have to use the Copy mode service. Do not rely on cursor style!

Reply:
Will make the changes accordingly.